finetune/base_models/textcnn/model.py

```diff
@@ -3,37 +3,76 @@
 
 from finetune.base_models import SourceModel
 from finetune.base_models.gpt2.encoder import GPT2Encoder
+from finetune.base_models.bert.roberta_encoder import RoBERTaEncoderV2
 from finetune.base_models.textcnn.featurizer import textcnn_featurizer
-from finetune.util.download import GPT2_BASE_URL, FINETUNE_BASE_FOLDER
+from finetune.util.download import (
+    GPT2_BASE_URL,
+    ROBERTA_BASE_URL,
+    FINETUNE_BASE_FOLDER,
+)
+
+KERNEL_SIZES = [2, 4, 8]
+TEXTCNN_BASE_PARAMS = {
+    "n_embed_featurizer": 768,
+    "n_layer": 1,
+    "n_epochs": 8,
+    "keep_best_model": False,
+    "early_stopping_steps": None,
+    "val_size": 0,
+    "chunk_long_sequences": False,
+    "num_layers_trained": 1,
+    "kernel_sizes": KERNEL_SIZES,
+    "num_filters_per_size": 256,
+    "n_embed": len(KERNEL_SIZES) * 256,
+    "act_fn": "gelu",
+    "train_embeddings": True,
+    "lr": 2e-3,
+    "seq_num_heads": len(KERNEL_SIZES) * 2,
+    "permit_uninitialized": r"conv[0-9]+",
```

Comment:
we should add max length in here?

Reply:
Do you think it would be better to have that here or in customv2? I realize I should also specify the predict batch size (in one of those two places)